test/test.env

```diff
@@ -1,2 +1,2 @@
 COMPANY_PROFILE_API_KEY=API_KEY
-LOG_LEVEL=silent
```

Comment:
I thought the 'silent' was a default for tests on purpose?

Reply:
yes, this should have not been included, reverted